tests/blackbox/testlib/dbus.py

```diff
@@ -466,3 +466,10 @@ def get_report(report_name):
             StratisDbus._REPORT_IFACE,
         )
         return iface.GetReport(report_name, timeout=StratisDbus._TIMEOUT)
+
+    @staticmethod
+    def close_connection():
+        """
+        Close bus connection.
+        """
+        StratisDbus._BUS.close()
```

Comment:
You'll need to actually reset the connection. If you just close it, any subsequent D-Bus method calls will fail. You'll need to reinitialize it too.